nominees/global-open-facility-registry.json

```diff
@@ -3,28 +3,32 @@
   "aliases": [
     "GOFR"
```

Comment:
@nathanbaleeta, could we add an alias for Global Open Facility Registry?

Reply:
@conradsp the `project name` field is already `Global Open Facility Registry`. So are you implying to switch `GOFR` to become the project name and then `Global Open Facility Registry` becomes an alias?

Reply:
@nathanbaleeta - Sorry, I missed that. I'll approve the PR.